gatsby/whatisgatsby.rst

```diff
@@ -19,3 +19,11 @@ with modern best-practices and tools:
 - One-directional data flow
 - ES6 + ES7 automatically transpiled 
 - PWA
+
+What makes GatsbyJS so popular is the simplicity of setting up a project using the latest and popular technologies 
+which pulls data from almost any source at all and generates a static site, just by whipping together a couple of
+Gatsby plugins together or even easier with a starter.
+
+That was why it's so popular with developers, but it even has a lot of benefits practically too! Static sites mean
+you don't need to worry about complex server setup, maintenance and scaling, you can just lift them into a CDN and
+make the site available as quick as possible to the users wherever they are. 
```

Comment:
These two paragraphs are loaded with terms that make it opinionated: "simplicity", "latest", "popular", etc. Try writing it without adjectives and adverbs to lose that tone, which should bring out what a developer who uses Gatsby can actually do with it.

Reply:
better for me

Reply:
Some remarks :)

Please remove "simplicity", and " even easier".

These is "bad" style because:

- this maybe is right for you, but that does not mean that it is "simple" for others.

Please try to think about your audience, for which audience is this training. ?

If  you are a person new to all this, this could be misleading and demotivating.

Also, I can tell you from personal experience with Gatsby that depending on what you are doing, the " simplicity of setting up a project" is sometimes gone in seconds :)

Further the road, please consider that we all have different backgrounds and knowledge of the english language.

Please do not use "That's" use "that is", the same goes for did not, do not, etc, etc.

And last, please try to use one sentence per line, in your example that is not always the case :) 

Tip: Re-watch the video from my talk last year at the conference, the talk has lots of examples about "do's and don'ts" for writing documentation/training :)

Thanks and have a nice weekend !

Reply:
@stevepiercy, sorry for being late to the party, I am traveling and have unstable internet :)

Reply:
@svx should we change [this guideline](https://docs.plone.org/about/contributing/documentation_styleguide.html#tone) in a separate issue or PR?

> Use the active voice whenever possible, and _contractions_ and pronouns are acceptable (in particular, the use of you in regards to the reader).

That's the only place I found in the guide with the word "contraction", and it seems out of place. Maybe this would be better?

```
Use the active voice whenever possible.
Pronouns are acceptable (in particular, the use of "you" in regards to the reader).

Avoid contractions, and spell out the words. For example, use "do not" instead of "don't".
See `Preferred Word Choice <https://docs.plone.org/about/contributing/word_choice.html#preferred-word-choice>`_
```

...and then add contractions as things to avoid under Preferred Word Choice? There's too many to list.

Reply:
@stevepiercy good idea, I have that on my list but I always forget to do it :)

Reply:
Oh I was unaware of this, will go through this properly before moving on to writing documentation for my part of the training. Thanks for letting me know early, this saves me time!

Reply:
Oh I was unaware of this, will go through this properly before moving on to writing documentation for my part of the training. Thanks for letting me know early, this saves me time!